www/generate_tutorial/src/input/tutorial.md

```diff
@@ -161,7 +161,7 @@ main =
     Stdout.line "There are \(total) animals."
 ```
 
-Now run `roc dev` again. This time the "Downloading …" message won't appear; the file has been cached from last time, and won't need to be downloaded again.
+Now run `roc dev` again. This time the "Downloading ..." message won't appear; the file has been cached from last time, and won't need to be downloaded again.
```

Comment:
Did the ellipsis symbol cause any issues?

Reply:
Yes, it crashes the parser from memory. Was going to put an issue in to track it.